assignment_02/app/Contracts/Services/MajorServiceInterface.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App\Contracts\Services;
+
+interface MajorServiceInterface
+{
+    public function getAllMajors();
+
+    public function create(array $data);
+
+    public function destroy(int $id);
+    public function getMajorById(int $id);
```

Comment:
If you break with empty line, then break all functions with empty line.
If your don't use new empty line, then remove new lines.
Make all the same format.